qabelbox/src/main/res/layout/item_chat_message_in.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:id="@+id/checkMessageIn"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center_horizontal"
+    android:orientation="horizontal">
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="25sp"
+        android:layout_marginTop="10sp"
+        android:background="@drawable/chat_in_left"/>
+    <RelativeLayout
```

Comment:
- pls try to avoid releativelayouts... this are the slowest layouts on android. better is a combination of frame and linear layout

- add image resources for chat items for all grafiksets from hdpi until xxxdpi